code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -105,22 +105,24 @@
 	span_xenowarning("We thrash about in a murderous frenzy!"))
 
 	X.face_atom(A)
-	var/sweep_range = 1
-	var/list/L = orange(sweep_range, X) // Not actually the fruit
-	var/victims = 0
-	var/target_facing
-	for(var/mob/living/carbon/human/H in L)
-		if(victims >= 3) //Max 3 victims
-			break
-		target_facing = get_dir(X, H)
-		if(target_facing != X.dir && target_facing != turn(X.dir,45) && target_facing != turn(X.dir,-45) ) //Have to be actually facing the target
+
+	var/list/atoms_to_ravage = list()
+	atoms_to_ravage += get_step(owner, owner.dir).contents
```

Comment:
Var/list/atomstoravage = get_step.contents ?

Reply:
Also type cast the list for clarity

Reply:
Sorry, get_steps.contents.copy.

If it is not possible, then eh it is micro op so not that important